Source/Csla.Channels.RabbitMq/RabbitMqPortal.cs

```diff
@@ -78,21 +48,15 @@ private void InitializeRabbitMQ()
     {
       if (Connection == null || DataPortalUri == null || Channel == null || DataPortalQueueName == null)
       {
-        Console.WriteLine($"Initializing connection to {DataPortalUrl}");
-        DataPortalUri = new Uri(DataPortalUrl);
-        var url = DataPortalUri;
-        if (url.Scheme != "rabbitmq")
-          throw new UriFormatException("Scheme != rabbitmq://");
-        if (string.IsNullOrWhiteSpace(url.Host))
-          throw new UriFormatException("Host");
-        DataPortalQueueName = url.AbsolutePath.Substring(1);
-        if (string.IsNullOrWhiteSpace(DataPortalQueueName))
-          throw new UriFormatException("DataPortalQueueName");
-
-        var factory = new ConnectionFactory { HostName = url.Host };
-        if (url.Port < 0)
-          factory.Port = url.Port;
-        var userInfo = url.UserInfo.Split(':');
+        Console.WriteLine($"Initializing connection to {DataPortalUri}");
+        if (DataPortalUri == null)
+          throw new NotSupportedException($"{nameof(DataPortalUri)} == null");
```

Comment:
That should be an `InvalidOperationException`. As far as I know `NotSupportedException` is only used in place where it's truly _not_ supported. Regardless what you are trying to do.